tendermint/src/rpc/id.rs

```diff
@@ -18,12 +26,16 @@ impl Id {
             .set_version(uuid::Version::Random)
             .build();
 
-        Id(uuid.to_string())
+        Id::Str(uuid.to_string())
     }
 }
 
 impl AsRef<str> for Id {
```

Comment:
Currently, this isn't used anywhere 🤔

Reply:
I believe so and I'd like to get rid of it. AsRef-ing a string (previous iteration) made sense, AsRef-ing this enum doesn't really.
I'm going to delete it and see if tests still succeed.

Reply:
Yeah, we didn't use it. Tests and build works.